files/zh-cn/web/http/headers/index.md

```diff
@@ -379,7 +379,7 @@ _[在此处](/zh-CN/docs/Glossary/CORS)了解更多关于 CORS。_
 - {{HTTPHeader("Signed-Headers")}} {{experimental_inline}}
   - : [`Signed-Headers`](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#rfc.section.5.1.2) 标头字段标识要包含在签名中的响应标头字段的有序列表。
 - {{HTTPHeader("Server-Timing")}}
-  - : 传达给定请求-响应周期的一个或多个指标和描述。
+  - : 传达给定请求 - 响应周期的一个或多个指标和描述。
```

Comment:
why this is different from line 59 🤔 

```suggestion
  - : 传达给定请求—响应周期的一个或多个指标和描述。
```

Reply:
Didn't notice the change on this line, the modification on line 59 was manually modified by myself (`autocorrect` just gave similar changes to this line).